docs/how-to/rocm-for-hpc/index.rst

```diff
@@ -0,0 +1,239 @@
+        - Open-source Navier-Stokes solver based on the spectral element method
+          targeting CPUs and accelerators that support :doc:`HIP <hip:index>`,
+          CUDA, or OpenCL. 
+
+      * - Computational chemistry
+        - `QUDA <https://github.com/amd/InfinityHub-CI/tree/main/quda>`_
+        - Library designed for efficient lattice QCD computations on
+          accelerators. It includes optimized Dirac operators and a variety of
+          fermion solvers and conjugate gradient (CG) implementations, enhancing
+          performance and accuracy in lattice QCD simulations.
+
+      * - Electronic structure
+        - `CP2K <https://github.com/amd/InfinityHub-CI/tree/main/cp2k>`_
+        - Versatile code for atomistic simulations across various systems:
+          solid state, liquid, molecular, periodic, material, crystal, and
+          biological. It supports multiple theory levels including DFTB, LDA,
+          GGA, MP2, RPA, semi-empirical methods, and classical force fields.
+          CP2K enables simulations such as molecular dynamics, metadynamics,
+          Monte Carlo, Ehrenfest dynamics, vibrational analysis, core level
+          spectroscopy, energy minimization, and transition state optimization
+          using NEB or dimer methods.
+
+      * - Quantum Monte Carlo Simulation
+        - `QMCPACK <https://github.com/amd/InfinityHub-CI/tree/main/qmcpack>`_
+        - Quantum Monte Carlo code designed for computing the electronic
+          structure of atoms, molecules, 2D nanomaterials, and solids. It
+          supports a wide range of materials, including metallic systems and
+          insulators. QMCPACK is optimized to perform effectively across various
+          computing platforms, from workstations to the latest supercomputers.
+          In addition to high performance, QMCPACK prioritizes code quality and
+          reproducibility.
+
+      * - Climate and weather
+        - `MPAS <https://github.com/amd/InfinityHub-CI/tree/main/mpas>`_
+        - Collaborative project between COSIM at Los Alamos and the National
+          Center for Atmospheric Research for developing atmosphere, ocean, and
+          other Earth-system simulation components for use in climate, regional
+          climate, and weather studies.
+
+      * - Benchmark
+        - `HPCG <https://github.com/amd/InfinityHub-CI/tree/main/hpcg>`_
+        - High Performance Conjugate Gradient Benchmark: a complement to the
+          High Performance LINPACK (HPL) benchmark. The computational and data
+          access patterns of HPCG are designed to closely match a broad set of
+          important applications not represented by HPL, and to incentivize
+          computer system designers to invest in capabilities that benefit the
+          collective performance of these applications.
+
+      * -
+        - `rocHPL <https://github.com/amd/InfinityHub-CI/tree/main/rochpl>`_
```

Comment:
Perhaps this row should go before the previous row.

Reply:
done